Tests/scripts/collect_tests/collect_tests.py

```diff
@@ -920,7 +919,7 @@ def _collect_single(self, path: Path) -> Optional[CollectionResult]:
             return self._collect_yml(path)
 
         elif file_type == FileType.REPUTATION:
-            tests = DEFAULT_REPUTATION_TESTS
+            tests = self.conf['reputation_tests']
             reason = CollectionReason.DEFAULT_REPUTATION_TESTS
```

Comment:
For clarity, please change the Enum value of `CollectionReason.DEFAULT_REPUTATION_TESTS` to `Incident type file changed, running reputation tests from conf.json['reputation_tests']`